src/main/java/com/softech/coursesvc/CourseList.java

```diff
@@ -0,0 +1,18 @@
+package com.softech.coursesvc;
+
+import com.softech.coursesvc.entity.CourseEntity;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.stereotype.Component;
+
+
+import java.util.List;
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Component
```

Comment:
Remove @Conponent annotaion from here... it is acting like our dto for trasferung list of courses... its not a spring bean like repository, service...

Reply:
Okay. I will do it.